tests/test_combobox.py

```diff
@@ -19,3 +19,7 @@ def test_set_selected(self):
         value = 0
         self.combobox.setSelected(value)
         self.assertEqual(self.combobox.selected(), value)
+
+    def test_can_pass_items_to_constructor(self):
+        """Tests user can pass items to the constructor of the combobox."""
+        combobox = Combobox(['Blue', 'Yellow', 'Red'])
```

Comment:
:) Good idea. But I think it's better to avoid such things until we find good ways to write tests. I really like your idea, but it will be confusing for other people.